test/DurableTask.SqlServer.Tests/Utils/TestInstance.cs

```diff
@@ -71,9 +71,13 @@ public async Task<OrchestrationState> WaitForStart(TimeSpan timeout = default)
             OrchestrationStatus expectedStatus = OrchestrationStatus.Completed,
             object expectedOutput = null,
             string expectedOutputRegex = null,
-            bool continuedAsNew = false)
+            bool continuedAsNew = false,
+            bool doNotAdjustTimeout = false)
         {
-            AdjustTimeout(ref timeout);
+            if (!doNotAdjustTimeout)
```

Comment:
Needed to add this, otherwise we can't write test code that waits for timeouts (like I do in my new test) when a debugger is attached.